flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/dataformat/BinaryRow.java

```diff
@@ -55,7 +55,7 @@
 public final class BinaryRow extends BinaryFormat implements BaseRow {
 
 	public static final boolean LITTLE_ENDIAN = (ByteOrder.nativeOrder() == ByteOrder.LITTLE_ENDIAN);
-	private static final long FIRST_BYTE_ZERO = LITTLE_ENDIAN ? 0xFFF0 : 0x0FFF;
+	private static final long FIRST_BYTE_ZERO = LITTLE_ENDIAN ? 0xFFFFFFFFFFFFFF00L : 0x00FFFFFFFFFFFFFFL;
```

Comment:
Actually I want use ~0xFFL instead. But I'm not sure what the perform in big endian.

Reply:
`~0xFFL` and `~(0xFFL << 56L)`?

Reply:
After test, it seems to that have the same expression with 0xFFFFFFFFFFFFFF00L and 0x00FFFFFFFFFFFFFFL, I am going to use the short expression.